integration/exec/workingdir_test.go

```diff
@@ -0,0 +1,62 @@
+package exec
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/api/types/strslice"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExecWithWorkingDir(t *testing.T) {
```

Comment:
We shouldn't write an integration test for option. How about just making this `TestExec()` and we can add a bunch of fields to this single test.

Reply:
Do you mean you prefer a single, huge test to check all options and their correct behaviour ? Won't this make it harder to diagnose a build failure with a clear cause ?

Reply:
It shouldn't be any harder if the failure messages are written correctly. 

If there are options that conflict with each other, or that significantly change the behaviour of other options then multiple tests might be necessary, but I don't think that's the case here. Most of the options are independent, and we don't need to run 5 slow tests to check each independent option.

Reply:
right, but testing workdir require to set a custom command executed, while testing other options like env will require to run `/bin/env`, etc...

Reply:
fair point. For those two cases I think `env` works for both, since `PWD` should be set. Is that right?

Reply:
hum, I don't this PWD is a standard env variable, not to confuse with $PWD (alias for `$(pwd)`)

Reply:
implemented as https://github.com/moby/moby/pull/35661/commits/57c5631ba7541d67d35e7c0a1f23c42a222ccb55